.github/sentry/__tests__/response-cache.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { createYoga } from 'graphql-yoga'
+import { useResponseCache } from '@graphql-yoga/plugin-response-cache'
+
+it('cache a query operation', async () => {
```

Comment:
I think this file is in a wrong place :D

Reply:
accidentally committed it lol